### PR TITLE
New package: Romanitho.Winget-AutoUpdate version 1.20.0

### PR DESCRIPTION
**Pull request by G0rocks**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152510)

### CODE REVIEW

**Comment by G0rocks** [comment]
Made with wingetcreate

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-05-07 04:56:25.634 [CLI ] ShellExecute installer failed: 4294967295

Automated error analysis suggests -1978335226 may mean Install successful but system restart may be required.

(Automated response - build 880.)

**Comment by G0rocks** [comment]
Yeah I tried checking the source (https://github.com/Romanitho/Winget-AutoUpdate/blob/main/Sources/WAU/Winget-AutoUpdate/Winget-Install.ps1) for this installation error code but I didn't find it. No clue what it's about.
Tagged the developer of WAU via issue: https://github.com/Romanitho/Winget-AutoUpdate/issues/625
@Romanitho if you (or anybody else) know what's going on and how to fix it then that would be great, otherwise I'm closing this PR

**Comment by stermic2** [comment]
@G0rocks I have to ask... The WinGet Auto-Update Configurator seems to be an installer, with extra steps, for WinGet Auto-Update.

I may be wrong, (there might not be an alternative) but installing an installer seems to be inferior to just installing with whatever extra parameters the configurator might be pushing.

Asking out of curiosity, I'm looking into WinGet Auto-Update for Enterprise use and need to learn.  I'm concerned that an installer that downloads files after 'installing' also circumvents the security advantages of Winget.

You can tell me bluntly if I'm not making sense. I'm just beginning my research on the matter.

File: manifests/r/Romanitho/Winget-AutoUpdate/1.20.0/Romanitho.Winget-AutoUpdate.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Romanitho.Winget-AutoUpdate
+PackageVersion: 1.20.0
+InstallerType: zip
+NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/r/Romanitho/Winget-AutoUpdate/1.20.0/Romanitho.Winget-AutoUpdate.installer.yaml:7]:
```suggestion
NestedInstallerType: portable
```

**Comment by AndrewDemski-ad-gmail-com** [comment]
> Automatic Validation ended with:
> 
> > Installation failed with exit code -1978335226
> > 2024-05-07 04:56:25.634 [CLI ] ShellExecute installer failed: 4294967295
> 
> Automated error analysis suggests -1978335226 may mean Install successful but system restart may be required.
> 
Far from it, APPINSTALLER_CLI_ERROR_SHELLEXEC_INSTALL_FAILED was returned for a fully scripted product and winget does not support those yet.
https://github.com/microsoft/winget-cli/blob/5fe31fc8710514f0f3f08d18d8043c43352e3f73/schemas/JSON/settings/settings.schema.0.2.json#L126-L137

About WAU
There is a ton of PS1 files, 1 VBS file, 1 LNK file and good old ServiceUI.exe, the installation procedure requires running PS1 file with some additional/optional parameters. 
Product could be installed either from scripted install action in deployment type in SCCM application or from intunewin file with little to no effort.
The installation logic is invoked only once and updates are pulled directly from github by dedicated scheduled task.

Moving/conversion of the update flow away from scripted solution backed by GitHub to winget pkgs repo was most likely not discussed with author of the WAU ( [@Romanitho](https://github.com/Romanitho) ).

**Comment by AndrewDemski-ad-gmail-com** [comment]
One last thing, 
how was the Microsoft GitHub Policy Service / license/cla singed if WAU author was not contacted/involved n the process of importing WAU into pkgs repository?

**Pull request closed without merging**